src/main/java/org/fundacionjala/coding/cynthia/SpinWord.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.cynthia;
+
+
+import java.util.StringJoiner;
+
+/**
+ * clase spin words.
+ */
+public class SpinWord {
+
+    public static final int LIMIT = 5;
```

Comment:
Since that this constant will not be used in another classes.
It should be private